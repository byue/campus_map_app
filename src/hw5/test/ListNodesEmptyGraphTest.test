# Tests that a new graph can be created, and
# no nodes are listed for an empty graph.

# Create a graph
CreateGraph graph1

# Print the nodes in the graph
ListNodes graph1
