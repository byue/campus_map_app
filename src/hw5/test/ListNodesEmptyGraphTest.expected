# Tests that a new graph can be created, and
# no nodes are listed for an empty graph.

# Create a graph
created graph graph1

# Print the nodes in the graph
graph1 contains: